src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int BALLS_COUNT = 6;
+
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball[] ball = new Ball[BALLS_COUNT];
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < BALLS_COUNT; i++) {
+            ball[i] = lottery.getRandomBall();
+            System.out.println(ball[i].toString());
```

Comment:
```suggestion
            System.out.println(ball[i]);
```

`toString()` method will be called "under the hood"